### PR TITLE
Add visualization preference panel

### PR DESCRIPTION
**Pull request by chenghuzi**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## What does this implement/fix? Explain your changes.

Add a preference panel for users to select which charts to show or hide. The preference will be loaded from and stored to the local storage.

### CODE REVIEW

File: optuna_dashboard/static/components/StudyDetail.tsx [hunk 1]
```diff
@@ -15,9 +17,10 @@ import {
   IconButton,
   Select,
   MenuItem,
+  FormGroup,
 } from "@material-ui/core"
-import { Home, Cached } from "@material-ui/icons"
-
+import { Home, Cached, Settings } from "@material-ui/icons"
+import CloseIcon from "@material-ui/icons/Close"
```
Comment (c-bata) [inline optuna_dashboard/static/components/StudyDetail.tsx:23]: How about importing `CloseIcon` component like L22?

**Review by c-bata — APPROVED** [review]
Thank you for your pull request. PR looks almost perfect 👍 

LGTM with nits!

File: optuna_dashboard/static/components/StudyDetail.tsx (hunk 1 above)
Comment (chenghuzi) [reply optuna_dashboard/static/components/StudyDetail.tsx:23]: It's not included in the corresponding type definition files, so we may have to keep this line.
```
ERROR in /Users/huzi/Workspace/github/optuna-dashboard/optuna_dashboard/static/components/StudyDetail.tsx
./optuna_dashboard/static/components/StudyDetail.tsx 22:33-42
[tsl] ERROR in /Users/huzi/Workspace/github/optuna-dashboard/optuna_dashboard/static/components/StudyDetail.tsx(22,34)
      TS2305: Module '"@material-ui/icons"' has no exported member 'CloseIcon'.
 @ ./optuna_dashboard/static/components/App.tsx 5:0-44 12:119-130
 @ ./optuna_dashboard/static/index.tsx 3:0-39 5:24-27
```

Comment (c-bata) [reply optuna_dashboard/static/components/StudyDetail.tsx:23]: I see! It may be an issue of type stub file. Anyway, I merged the pull request.

**Pull request merged**